js/apps/system/_admin/aardvark/APP/foxxes.js

```diff
@@ -483,25 +483,6 @@ router.post('/download/nonce', function (req, res) {
   Creates a cryptographic nonce that can be used to download the service without authentication.
 `);
 
-anonymousRouter.get('/download/zip', function (req, res) {
-  const nonce = decodeURIComponent(req.queryParams.nonce);
-  const checked = nonce && crypto.checkAndMarkNonce(nonce);
```

Comment:
BTW. I wonder whether the nonce feature is broken in general or only here

Reply:
removed because don't need a nonce since `_api/` routes are authenticated. In fact, we can remove the download/nonce endpoint above this as well